out/automation/util/early-game-start-hack.js

```diff
@@ -52,4 +52,21 @@ export async function main(ns) {
 	ns.exec("/automation/util/weaken.js", server, weakenThreads, target);
 	ns.exec("/automation/util/grow.js", server, growThreads, target);
 	ns.exec("/automation/util/hack.js", server, hackThreads, target);
+}
+
+export function autocomplete(data, args) {
+	data.flags([
+		["server", null],  // the server that will run the scripts
+		["target", null],  // the target server
+		["verbose", false], // print errors if launching the scripts fails
+	])
+	if (args.length === 1 || args.length === 2){
+		return data.servers
+	}
+	const options = {
```

Comment:
How does declaring this constant affect anything?

Reply:
This ensures the first or second argument passed will autocomplete out of all server names.

Reply:
it does not! good call. I'd gotten mixed up, as early-game has no flags.